processing/src/test/java/io/druid/segment/data/IncrementalIndexTest.java

```diff
@@ -436,6 +537,8 @@ public void run()
         );
       }
     }
+
+    index.close();
```

Comment:
these should be done in a closer or finally block or something to ensure they actually get closed.